src/common.rs

```diff
@@ -6,7 +6,9 @@
 #![allow(missing_docs)]
 
 mod sealed {
-    pub trait Sealed {}
+    /// A sealed trait which [sealed trait](https://predr.ag/blog/definitive-guide-to-sealed-traits-in-rust/) which stops this trait
```

Comment:
I think this reads:
A sealed trait which sealed trait which..., which doesn't seem right

Reply:
whoops.